addons/block_code/simple_nodes/simple_sound/simple_sound.gd

```diff
@@ -0,0 +1,33 @@
+@tool
+class_name SimpleSound
+extends AudioStreamPlayer
+
+var soundstream: AudioStream = load("res://addons/block_code/simple_nodes/simple_sound/score.ogg")
+
+
+func _ready():
+	stream = soundstream
+
+
+func get_class():
+	return "SimpleSound"
+
+
+static func get_exposed_properties() -> Array[String]:
+	return ["stream"]
+
+
+static func get_custom_blocks() -> Array[BlockCategory]:
+	var b: Block
+
+	# Play the sound
+	var sound_list: Array[Block] = []
+	b = CategoryFactory.BLOCKS["statement_block"].instantiate()
+	b.block_type = Types.BlockType.EXECUTE
+	b.block_format = "Play the sound, Volume dB {db: INT}"
+	b.statement = "volume_db=db\n" + "play()"
```

Comment:
Also, make sure the parameter to format is surrounded by curly braces `{}` or it won't be replaced by the value in the block.
```suggestion
	b.statement = "volume_db={db}\n" + "play()"
```

Reply:
cool!  thanks!